spec/requests/users_spec.rb

```diff
@@ -105,4 +105,47 @@
       end
     end
   end
+
+  describe 'GET index' do
+    context 'when user is authenticated' do
+      let!(:users) { create_list(:user, 8) }
+      let!(:user) { users.first }
```

Comment:
You can move this variables definition outside contexts, and put it inside the whole describe, so you don't have to repeat the code in both contexts